src/libs/collect.c

```diff
@@ -1391,17 +1392,20 @@ static void combo_changed(GtkComboBox *combo, dt_lib_collect_rule_t *d)
      || property == DT_COLLECTION_PROP_ISO)
   {
     gtk_widget_set_tooltip_text(d->text, _("type your query, use <, <=, >, >=, <>, =, [;] as operators"));
+    dt_gui_add_help_link(d->text, "collect_images.html#collect_images_usage");
```

Comment:
You don't need to set the help text in this function, just do it once in `gui_init()` when the widgets are created.